Content.Shared/_DV/Xenoarchaeology/BUI/AnalysisConsoleBoundUserInterfaceState.cs

```diff
@@ -0,0 +1,20 @@
+using Content.Shared.Flash.Components;
+using Content.Shared.Psionics.Glimmer;
+using Content.Shared.Xenoarchaeology.Equipment.Components;
+using Robust.Shared.Serialization;
+
+namespace Content.Shared._DV.Xenoarchaeology.BUI;
+
+// DeltaV
+[Serializable, NetSerializable]
+public sealed class AnalysisConsoleBoundUserInterfaceState : BoundUserInterfaceState
+{
+    public float Mult { get; }
+    public float Ratio { get; }
```

Comment:
A comment here could save readers some time figuring out what these are related to.

```suggestion
    // Point multiplier from glimmer, and glimmer-generated-per-points ratio. See these variables in ArtifactAnalyzerComponent
    public float Mult { get; }
    public float Ratio { get; }
```